refinery/ui/source/js/file-browser/services/factory.js

```diff
@@ -197,7 +197,7 @@
         assayFilesTotalItems.count = response.nodes_count;
 
         // Not concat data when under minimun file order, replace assay files
-        if (assayFilesTotalItems.count < maxFileRequest && params.offset === 0) {
+        if (assayFilesTotalItems.count < maxFileRequest || params.offset === 0) {
```

Comment:
This makes sense, but I'm not sure I understand when this branch would be taken: If the total files is less the the max files per request, this should only be hit on the initial page load, and there won't be any loading-on-scroll to worry about?